policies/all-tasks.yaml

```
@@ -15,7 +15,7 @@ sources:
       required_task_results:
         # Certain EC rules rely on the presence of this result when validating an image.
         - task: clair-scan
-          result: CLAIR_SCAN_RESULT
+          result: SCAN_OUTPUT
```

With the new policy, we want to configure that for 0.1 the result name is `CLAIR_SCAN_OUTPUT`, and for all other versions it should be `SCAN_OUTPUT`, something like this:

```suggestion
          result: CLAIR_SCAN_OUTPUT
          version: 0.1
        - task: clair-scan
          result: SCAN_OUTPUT
```